source/GameData.cpp

```diff
@@ -553,8 +553,12 @@ void GameData::Change(const DataNode &node)
 // that a change creates or moves a system.
 void GameData::UpdateNeighbors()
 {
-	for(auto &it : systems)
+	for(auto &it : systems) {
+		// Skip systems that have no name.
+		if(it.first.empty() || it.second.Name().empty())
```

Comment:
I'm not sure `it.first` can ever be empty, as that would imply that the game called `GameData::Systems().Get("")`

Reply:
I agree, it shouldn't be.  I copied the condition from another place in the source that also used it, and it felt like the extra paranoia wouldn't hurt anything.